web/src/app/schedules/ScheduleOverrideForm.js

```diff
@@ -38,163 +39,159 @@ const query = gql`
   }
 `
 
-const styles = theme => ({
+const useStyles = makeStyles({
   tzNote: {
     display: 'flex',
     alignItems: 'center',
   },
 })
 
-@connect(state => ({ zone: urlParamSelector(state)('tz', 'local') }))
-@withStyles(styles)
-export default class ScheduleOverrideForm extends React.PureComponent {
-  static propTypes = {
-    scheduleID: p.string.isRequired,
+export default function ScheduleOverrideForm(props) {
+  const { add, remove, errors, scheduleID, value, ...formProps } = props
```

Comment:
Since errors isn't required, we should have `errors=[]` so that it defaults to an empty list